discovery-provider/prod.env

```diff
@@ -23,7 +23,7 @@ audius_elasticsearch_url='http://elasticsearch:9200'
 audius_elasticsearch_search_enabled='true'
 audius_enable_rsyslog=false
 audius_eth_contracts_registry=0xd976d3b4f4e22a238c1A736b6612D22f17b6f64C
-audius_eth_token_address=0x5375BE4c52fA29b26077B0F15ee5254D779676A6
+audius_eth_token_address=0x18aAA7115705e8be94bfFEBDE57Af9BFc265B998
```

Comment:
is this needed? unsure how this is related to the PR?

Reply:
it is, in order to init the eth contracts right (and reference the token), this value needs to be correct :)